MutationObserver.js

```diff
@@ -464,7 +501,7 @@ window.MutationObserver = window.MutationObserver || (function(undefined) {
                      */
                     elestruct.attr = reduce($target.attributes, function(memo, attr) {
                         if (!config.afilter || config.afilter[attr.name]) {
-                            memo[attr.name] = attr.value;
+                            memo[attr.name] = getAttributeValue($target, attr);
```

Comment:
I briefly looked at the code and I think `attributeOldValue` should be fine now as we always call `getAttributeValue` when we are putting to the cache and when we getting a new value. But it's better if you do a proper testing anyway